src/sagas/admin.js

```diff
@@ -125,30 +181,46 @@ export function* issueCertificate({ payload }) {
     const issueMsg = contract.methods.issue(certificateHash);
     const gasPrice = (yield web3.eth.getGasPrice()) * 5;
     const gasLimit = (yield issueMsg.estimateGas()) * 2;
+    const transactionCostInEthers = web3.utils.fromWei(
+      (gasPrice * gasLimit).toString(),
+      "ether"
+    );
 
-    const txHash = yield sendTxWrapper({
-      txObject: issueMsg,
-      gasPrice,
-      gasLimit,
-      fromAddress
-    });
+    if (accountBalance >= transactionCostInEthers) {
```

Comment:
Note that accountBalance is a string in some components, which probably may cause issue here if not fixed